integration-tests/client/chainlink_k8s.go

```diff
@@ -0,0 +1,125 @@
+// Package client enables interaction with APIs of test components like the mockserver and Chainlink nodes
+package client
+
+import (
+	"fmt"
+	"os"
+	"regexp"
+
+	"github.com/rs/zerolog/log"
+
+	"github.com/smartcontractkit/chainlink-env/environment"
+)
+
+type ChainlinkK8sClient struct {
```

Comment:
I can't say it's decoupling at all, we just extended the client because of 2 fields and 2 wrong methods.
But tbh it doesn't matter since `k8s` stuff will going to change in the future and it'd be less `k8s` stuff when we move to docker. Then we clean it up properly.

Reply:
@skudasov what do you suggest then? Feel free to work on this branch to improve it. Thanks!

Reply:
I can agree with Adam's post
```
Is it necessary to create a separate client specifically for K8s? As far as I can tell, the only K8s-specific operation we do on the nodes once launched is the upgrade path. The rest is just plain old REST calls. The only thing to change for 99% of our use cases should just be the URL.
```
But in my opinion it doesn't matter right now, if you need it, let's merge @lukaszcl